DEPLOY.md

```diff
@@ -9,10 +9,11 @@ Travis-ci will push tagged commits to PyPI. Here are the steps:
 Next version number? (previous: 'XXX')
 ```
 
-2. Give the new version number based on previous version (see semantic release)
+2. Give the new version number based on previous version (Use semantic versioning)
 
-3. Now the push will be on hold until you press Enter. Manually inpect the changelog (`doc/changes.rst`) to make changes if necessary. Once you are sure, go back and press Enter. 
+3. Create a new Github [release](https://github.com/PyGithub/PyGithub/releases) from the tag that'sjust committed, with the same release note from `doc/changes.rst`. This step is the hook that will trigger the workflow. (also needed for some web spiders for changelog parsing)
```

Comment:
"that'sjust" => "that has just been committed" to correct the missing space and I think it scans better.

Reply:
You can use the suggest feature